src/app/components/CourseDetailHeader/CourseDetailHeader.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styles from "./CourseDetailHeader.module.css";
+import ClassBallDoubleDigit from "../../assets/icons/ClassBallDoubleDigit";
+import ClassBallNo from "../../assets/icons/ClassBallNo";
+import ClassBallSingleDigit from "../../assets/icons/ClassBallSingleDigit";
+import RoundImage from "../RoundImage/RoundImage";
+
+function CourseDetailHeader(): JSX.Element {
+  return (
+    <div className={styles.header}>
+      <section className={styles.header__image}>
+        <RoundImage imageSrc="/images/basketball.png" />
+      </section>
+      <section className={styles.header__text}>
+        <h1>Basketball</h1>
+        <span className={styles.header__text__body}>Leitung: Peter Lustig</span>
+        <section className={styles.header__text__classes}>
+          Jg
```

Comment:
Jahrgang? I am not sure if this is self explaining.

Reply:
Grrr! :-) You are right! I changed it. Thank you!